src/__experimental__/components/input/input.style.js

```diff
@@ -9,7 +9,8 @@ const StyledInput = styled.input`
   flex-grow: 1;
   font-size: ${({ theme }) => theme.text.size};
   outline: none;
-  width: 30px;
+  text-align: ${({ align }) => align}
```

Comment:
align prop needs a default value otherwise this is breaking the css for other components

Reply:
I've changed this slightly this morning, so `text-align` is only used if the `align` prop is used.